Sources/TestsCommon/CommonTestUtils.swift

```diff
@@ -188,7 +188,6 @@ public enum TestTopologyConfiguration: String, Decodable {
             helloReply["isreplicaset"] != true
         {
             self = .single
-            // TODO: SWIFT-1319: eventually, we should be able to just check for serviceId here.
```

Comment:
as discussed offline, serviceId only shows up in the initial handshake, so we can't check for it here and need to still rely on the environment variable.